hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import json
+
+from hmalib.common import get_logger
+from hmalib.models import MatchMessage
+
+logger = get_logger(__name__)
```

Comment:
doesn't look like we use this yet? maybe add an event received log for now?

Reply:
I think this will probably get used in future, so fine leaving it in, and impl below is definitely not filled in.